WIP:  Initial DRAFT of Care&Feeding doc for Arcade (#4429)
dotnet/arcade: Documentation/Policy/ArcadeContributorGuidance.md

> * All discussions (arguments) must be civil, respectful, productive, and above all - kind. The focus must always be on the topic, not individuals.

Reviewer: ```suggestion
* All discussions (arguments) [must be civil, respectful, productive, and above all - kind](https://microsoft.github.io/codeofconduct/). The focus must always be on the topic, not individuals.
```

Arcade already states its adoption of the Microsoft Code of Conduct in the readme, but this is a good place to reinforce the commitment.

* Work in 'partnership' (shoulder to shoulder) as opposed to 'us' vs. 'them'.

## Arcade Core Team
### Expectations
- Answer questions, provide guidance, and generally be helpful and responsive.
- Pay attention.  Monitor the activity in the repo and take the appropriate action when necessary
- Triage, including old PRs (see section below for more on this) 

### Ownership
The current owner for dotnet/arcade is Mark Wilkie <mawilkie@microsoft.com>.  The current point persons are:
- Alex Perovich <Alex.Perovich@microsoft.com>
- Jon Fortescue <Jonathan.Fortescue@microsoft.com>
- Michael Stuckey <Michael.Stuckey@microsoft.com>
- Ricardo Arenas <riarenas@microsoft.com>

## Getting Things Fixed
### Submit a PR
This is probably the most straight forward way.  If the PR might take some time and you want to get an early read first, filing an issue to get a discussion going, or emailing arcadewg@microsoft.com should get the insight you need.

### Filing Issues in Arcade

> All issues regarding our shared infrastructure. (Includes Arcade, darc, Maestro/dependency flow, and Helix) should be filed in the dotnet/arcade repo.  If there's a doubt, file in Arcade.  (As an aside, the private dotnet/core-eng repo is used by the engineering services team for internal facing work)

Reviewer: ```suggestion
All issues regarding our shared infrastructure (including Arcade, darc, Maestro/dependency flow, and Helix) should be filed in the dotnet/arcade repo.  If there's a doubt, file in Arcade.  (As an aside, the private dotnet/core-eng repo is used by the engineering services team for internal facing work.)
```

Reviewer: or simply...

```suggestion
All issues regarding our shared infrastructure, including Arcade, darc, Maestro/dependency flow, and Helix, should be filed in the dotnet/arcade repo. 
```

The default Arcade issue template should be used with every issue filed.  The template is *very* simple and has low overhead.

### Triage and Backlog Management
- New Arcade issues are triaged at least once a week.  
- Issues which have been marked via the template as blocking or causing unreasonable pain are looked at right away via FR ([First Responders](https://github.com/dotnet/core-eng/wiki/%5Bint%5D-First-Responders))

> - General "scrubs" happen at least once a yea.r

Reviewer: ```suggestion
- General "scrubs" happen at least once a year.
```
